packages/components/src/components/contract-card/contract-card-items/__tests__/contract-update-form.spec.tsx

```diff
@@ -1,9 +1,14 @@
 import React from 'react';
 import { configure, render, screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
-import { CONTRACT_TYPES, mockContractInfo, getCardLabels } from '@deriv/shared';
+import { CONTRACT_TYPES, mockContractInfo, getCardLabels, isMobile } from '@deriv/shared';
 import ContractUpdateForm from '../contract-update-form';
 
+jest.mock('@deriv/shared', () => ({
+    ...jest.requireActual('@deriv/shared'),
+    isMobile: jest.fn(() => false),
```

Comment:
I have to mock the deprecated `isMobile` in order to test the usage of ContractUpdateForm in mobile where it's rendering MobileWrapper. MobileWrapper is dependent on `isMobile`.